### DISCUSSION TITLE
Bug

### DISCUSSION THREAD

**Discussion by zecasa** [Chia-Network/chia-blockchain#10643] [Support]
https://user-images.githubusercontent.com/85630387/157606357-3459dacc-d3ce-46ee-8db0-a1399c1387a2.mp4

This problem has not been fixed for 10 days., I deleted the pool list but it still didn't fix. No 3 dots in the upper right corner.

**Comment by mariano54** [comment]
This will be fixed in the next version. I apologize for the issue.

**Reply by zecasa** [reply]
Thanks Mariano, good to see you here.
